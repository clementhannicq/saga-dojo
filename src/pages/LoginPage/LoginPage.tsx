import React, { SFC } from "react";
import LoginForm from "../../containers/LoginForm";

const LoginPage: SFC<{}> = () => <LoginForm />;

export default LoginPage as any;
