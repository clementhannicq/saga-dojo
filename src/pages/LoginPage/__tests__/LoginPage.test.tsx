import { shallow } from "enzyme";
import React from "react";
import LoginPage from "../LoginPage";

describe("Pages", () => {
  describe("LoginPage", () => {
    it("should render", () => {
      const wrapper = shallow(<LoginPage />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
