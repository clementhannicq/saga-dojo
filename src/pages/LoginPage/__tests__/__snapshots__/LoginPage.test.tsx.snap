// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages LoginPage should render 1`] = `<Connect(LoginForm) />`;
